eq-author-api/schema/tests/comments.test.js

```diff
@@ -91,6 +93,120 @@ describe("comments", () => {
     });
   });
 
+  it("should edit a comment on question page and then query that comment", async () => {
+    await createComment(ctx, {
+      pageId: createdQuestionPage.id,
+      commentText: "a new comment is created",
+    });
+
+    const queryNewComment = await queryComments(ctx, {
+      pageId: createdQuestionPage.id,
+    });
+    const commentId = queryNewComment.comments[0].id;
```

Comment:
I think instead of querying the component after it is created you can do

```js
	const newComment = await createComment(ctx, {
      pageId: createdQuestionPage.id,
      commentText: "a new comment is created",
    });
	const commentId = newComment.id
```

Reply:
ahh - yes , thats simpler - ta